### PR TITLE
Feat/notifications

### PR DESCRIPTION
**Pull request by matiu**

### CODE REVIEW

File: lib/server.js [hunk 1]
```diff
@@ -143,6 +145,25 @@ CopayServer.prototype._verifySignature = function(text, signature, pubKey) {
...
 };
 
+
+/**
+ * notify
+ *
+ * @param type
+ * @param data
+ */
+CopayServer.prototype.notify = function(type, data) {
```
Comment (isocolsky) [inline lib/server.js]: This should be named ._notify as it is an internal method

**Comment by matiu** [comment]
updated and ready for review.

File: lib/model/txproposal.js [hunk 2]
```diff
@@ -13,8 +13,10 @@ function TxProposal(opts) {
   opts = opts || {};
 
   this.version = VERSION;
-  this.createdOn = Math.floor(Date.now() / 1000);
-  this.id = ('000000000000' + this.createdOn).slice(-12) + Uuid.v4();
+
+  var now = Date.now();
+  this.createdOn = Math.floor(now / 1000);
+  this.id = ('00000000000000' + now).slice(-14) + Uuid.v4();
```
Comment (maraoz) [inline lib/model/txproposal.js]: I would extract this to:
```
var zeroPad = function(x, length) {
  return (Array(length).join('0') + x).slice(-length);
};
```
for clarity

Comment (matiu) [inline lib/model/txproposal.js]: will do, thanks.

**Comment by matiu** [comment]
udpated.

**Pull request merged**